multiversx_sdk_cli/localnet/config_default.py

```diff
@@ -15,6 +15,9 @@
     genesis_delay_seconds=10,
     rounds_per_epoch=100,
     round_duration_milliseconds=6000,
+    # For the purpose of the localnet, we'll have 3x for Supernova (by default).
+    rounds_per_epoch_in_supernova=300,
+    round_duration_milliseconds_in_supernova=2000,
```

Comment:
Shouldn't we have the actual duration of the round (600ms)?

Reply:
Good question!

I've chosen 2 seconds by default in order to demand less resources from the host machine (development machine).

All good?

Reply:
Fine with me. We should at least document that the localnet starts with a roundtime of 2s and we should show people where to change in case they want to run a 600ms round, replicating mainnet config.